test/test.html

```diff
@@ -30,7 +30,6 @@
 
   <!-- Include anything you want to test -->
   <script src="../lib/p5.js" type="text/javascript" ></script>
-  <script src="../src/lib/opentype.js" type="text/javascript" ></script>
```

Comment:
One note, that I should have left in the PR comment: Since opentype is included in the p5.js bundle we don't need to load it explicitly here. This should help us validate whether any changes made to address #781 cause any issues. (I suspect that there are issues with opentype as-is in AMD/Browserify situations, but I haven't proved that out.)